pandas/tools/util.py

```diff
@@ -173,7 +174,9 @@ def to_numeric(arg, errors='raise', downcast=None):
             values = arg.values
     elif isinstance(arg, (list, tuple)):
         values = np.array(arg, dtype='O')
-    elif np.isscalar(arg):
+    elif lib.isscalar(arg):
```

Comment:
import these from pandas.types.common (like everything else). you prob have to fix is_decimal to import also (it should be imported into pandas.types.inference, which pandas.types.common subseuqently imports). pandas.types.common is the internal API.